packages/template-explorer/src/index.ts

```diff
@@ -64,7 +64,7 @@ window.init = () => {
     // to debug with custom nodeTransforms
     delete persistedState.options?.nodeTransforms
     ssrMode.value = persistedState.ssr
-    Object.assign(compilerOptions, persistedState.options)
+    Object.assign({}, defaultOptions)
```

Comment:
Is this change intentional? It looks like it might have come from a bad merge of `main`.

Reply:
All changes to non-runtime code have been undone